src/components/NavItems/Assistant/AssistantApiHandlers/useAssistantApi.jsx

```diff
@@ -225,11 +225,17 @@ export default function assistantApiCalls() {
     return await callAsyncWithNumRetries(
       MAX_NUM_RETRIES,
       async () => {
-        const result = await axios.get(
-          assistantEndpoint +
-            "kinit/machine-generated-text" +
-            "?text=" +
-            encodeURIComponent(text), // max URL length is 2048 characters
+        // const result = await axios.get(
+        //   assistantEndpoint +
+        //     "kinit/machine-generated-text" +
+        //     "?text=" +
+        //     encodeURIComponent(text), // max URL length is 2048 characters
+        // );
```

Comment:
Suggest you just remove the old impl rather than leaving it commented out - if we need the old code back later it's in the git history.